src/AllMovieContainer.js

```diff
@@ -16,8 +16,12 @@ const AllMovieContainer = ({ movies }) => {
 
     return (
         <section className='all-movie-container'>
-            <h1>All Movies</h1>
-            {allMovies}
+            <div className='title'>
```

Comment:
Good work adding a div around the title and the movies cards so they can be styled separately